src/test/java/cloud/eppo/helpers/AssignmentTestCase.java

```diff
@@ -0,0 +1,47 @@
+package cloud.eppo.helpers;
+
+import cloud.eppo.ufc.dto.VariationType;
+import java.util.List;
+
+public class AssignmentTestCase {
+  private final String flag;
+  private final VariationType variationType;
+  private final TestCaseValue defaultValue;
+  private final List<SubjectAssignment> subjects;
+  private String fileName;
+
+  public AssignmentTestCase(
```

Comment:
Mostly not new code; ported over from Android implementation ([source](https://github.com/Eppo-exp/android-sdk/blob/main/eppo/src/androidTest/java/cloud/eppo/android/helpers/AssignmentTestCase.java))